resources/lang/en/beatmaps.php

```diff
@@ -298,4 +298,8 @@
         'C' => 'C',
         'D' => 'D',
     ],
+    'panel' => [
+        'playcount' => 'Playcount: :count',
```

Comment:
not sure if there's a need to include the counts in the tooltip when they're already on the panel